proxyclient/m1n1/hv.py

```diff
@@ -1086,10 +1086,7 @@ def rh(base, off, width):
             self.log(f"PMGR R {base:x}+{off:x}:{width} = 0x{data:x} -> 0x{ret:x}")
             return ret
 
-        if self.iodev == IODEV.USB0:
-            atc = "ATC0_USB"
-        elif self.iodev == IODEV.USB1:
-            atc = "ATC1_USB"
+        atc = "ATC{}_USB".format(int(str(self.iodev)[-1]))
```

Comment:
Let's use an f-string here, since that's the style the rest of the code uses. Also, it probably makes sense to use the numeric value of the iodev and subtract USB0; that avoids the string munging and will trivially scale to >10 ports.

Reply:
done and replaced another instance of the same string munging